applications/nrf5340_audio/src/modules/lc3_decoder/lc3_decoder_private.h

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright(c) 2018 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+#ifndef _LC3_DECODER_PRIVATE_H_
+#define _LC3_DECODER_PRIVATE_H_
+
+#include "lc3_decoder.h"
+
+#include "aobj_api.h"
+#include "amod_api.h"
+#include "amod_api_private.h"
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdint.h>
+#include <zephyr/kernel.h>
+#include "LC3API.h"
+
+#define CONFIG_LC3_DECODER_MAX_CHANNELS (2)
```

Comment:
Should be a Kconfig

Reply:
Again this was me thinking how I would use this API in anger and the decoder work will be coved in OCT-2568, so can be ignored for now. Will clean up.